common/scylla-image-post-start.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Scylla cloud image post start service
+After=scylla-server.service
+ConditionPathExists=!/etc/scylla/machine_image_post_start
+
+[Service]
+Type=oneshot
+# The ExecStartPre makes sure the scylla_post_start.py script 
+# will run only after scylla-server is up and running
+ExecStartPre=/bin/sleep 30
```

Comment:
I tried to find better way to do this (in systemd directive), but not really able to find. There is a way to do same using .timer, I want to avoid to do that for just waiting few sec.
Anyway, did you tried to run script without this delay? It didn't work?
If so, we might have a bug on scylla-server side, it should be able to connect when it becomes 'running' state.
But it is another issue, you can keep this line if it needed.